model-optimizer/unit_tests/extensions/back/ChangeOutputTypeAttributes_test.py

```diff
@@ -48,19 +48,19 @@ def test_range_out_of_fp16_min(self):
         self.assertRaises(Error, ChangeOutputTypeAttributes().find_and_replace_pattern, graph)
 
     def test_cast_correct_case(self):
-        input_data = np.array([0, 1000, 4, 9, 0])
+        input_data = mo_array([0, 1000, 4, 9, 0])
```

Comment:
I don't see these unit-test changes as necessary. more-over this could break previous test logic. mo_array is for the code and for conscious usage in tests. Please revert unit-tests/ changes

Reply:
I will try, but I think that will result in a lot of manual changes in unit_tests, because all unit_tests use bare np.array without thinking about types.